package.json

```diff
@@ -1,15 +1,11 @@
 {
   "name": "geostyler-cql-parser",
-  "version": "3.0.2",
+  "version": "4.0.0-next.1",
```

Comment:
should this be part of the `main` branch?

Reply:
No, this is currently published as a pre-release so I can use it for testing with the other repos.